vsintegration/src/FSharp.Editor/QuickInfo/Views.fs

```diff
@@ -93,28 +93,32 @@ type internal QuickInfoViewProvider
                     yield inl
             }
 
-        let createTextLinks () =
-            let tb = TextBlock(TextWrapping = TextWrapping.Wrap, TextTrimming = TextTrimming.None)
-            DependencyObjectExtensions.SetDefaultTextProperties(tb, formatMap.Value)
-            tb.Inlines.AddRange inlines
-            if tb.Inlines.Count = 0 then tb.Visibility <- Visibility.Collapsed
-            tb.Resources.[typeof<Documents.Hyperlink>] <- getStyle()
-            tb :> FrameworkElement
-            
-        { new IDeferredQuickInfoContent with member x.Create() = createTextLinks() }
+        let tb = TextBlock(TextWrapping = TextWrapping.Wrap, TextTrimming = TextTrimming.None)
+        DependencyObjectExtensions.SetDefaultTextProperties(tb, formatMap.Value)
+        tb.Inlines.AddRange inlines
+        if tb.Inlines.Count = 0 then tb.Visibility <- Visibility.Collapsed
+        tb.Resources.[typeof<Documents.Hyperlink>] <- getStyle()
+        tb
 
-    let empty = 
-        { new IDeferredQuickInfoContent with 
-            member x.Create() = TextBlock(Visibility = Visibility.Collapsed) :> FrameworkElement }
+    let wrap (tb: TextBlock) =
+        let maxWidth = formatMap.Value.DefaultTextProperties.FontRenderingEmSize * 40.0
```

Comment:
No, it's arbitrary. I wanted to make the max-width proportional to font size. I guess it's around 360 dips (device independent pixels) for 9pt size font.